cmd/tailscale/cli/network-lock.go

```diff
@@ -267,14 +267,78 @@ var nlAddCmd = &ffcli.Command{
 	},
 }
 
+var nlRemoveArgs struct {
+	noResigning bool
+}
+
 var nlRemoveCmd = &ffcli.Command{
 	Name:       "remove",
 	ShortUsage: "remove <public-key>...",
 	ShortHelp:  "Removes one or more trusted signing keys from tailnet lock",
 	LongHelp:   "Removes one or more trusted signing keys from tailnet lock",
-	Exec: func(ctx context.Context, args []string) error {
-		return runNetworkLockModify(ctx, nil, args)
-	},
+	Exec:       runNetworkLockRemove,
+	FlagSet: (func() *flag.FlagSet {
+		fs := newFlagSet("lock remove")
+		fs.BoolVar(&nlRemoveArgs.noResigning, "no-resigning", false, "do not resign signature which would be invalidated by removal of trusted signing keys")
```

Comment:
Would it be better to have a `-re-sign` option that defaults to true rather than having an inverted meaning boolean flag?

Reply:
Yeah that makes sense, but I had to lookup how to unset a default true flag in Go. I think it ends up being `--re-sign=false`